app/assets/javascripts/pdf_viewer/web/pdfjs-init.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// nb. After making changes to this file, the copy in the viewer/web/ dir
+// must be updated.
+
+// Listen for `webviewerloaded` event to configure the viewer after its files
+// have been loaded but before it is initialized.
+//
+// PDF.js >= v2.10.377 fires this event at the parent document if it is embedded
+// in a same-origin iframe. See https://github.com/mozilla/pdf.js/pull/11837.
+try {
+  parent.document.addEventListener('webviewerloaded', onViewerLoaded);
```

Comment:
Again this is just a question or check. I know your layout does not load turbolinks but seeing that you **require_tree .** the javascript directory in **app/assets/javascripts/pdf_viewer/base.js** do we need to wrap this call with the line below  or you are sure turbolinks is not loaded when **require _tree  .**:

      $( document ).on('turbolinks:load', function() {

Reply:
I will double check this.

Reply:
Since turbolinks isn't being loaded, and neither is JQuery I don't think this is necessary at all.  I'd have to require both of those things additionally and I am not sure why I would want to.  It's also worth noting that this is kinda a cul-de-sac in the app - there are no links or other elements belonging to the app itself, only what is supplied by Hypothes.is.